parser/adaptationset_element_parser.cpp

```diff
@@ -54,10 +54,16 @@ bool AdaptationSetElementParser::ParseStart(const std::string &element_name,
       elem->min_bandwidth_ = strtoul(attrib[i + 1], NULL, 10);
     } else IF_ATTRIB_MATCHES("maxBandwidth") {
       elem->max_bandwidth_ = strtoul(attrib[i + 1], NULL, 10);
+    } else IF_ATTRIB_MATCHES("width") {
```

Comment:
These attributes are parsed in RepresentationBase parser, no need to do it there, but you might want to check does it really work